### PR TITLE
Update Spike and Makeshift Knife recipe to be more reasonable.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Balance "Updates Spike and Makeshift Knife recipe to be more reasonable".

#### Purpose of change

To make the Makeshift knife recipe more sensible. 
#### Describe the solution

Change volume and weight of spike and makeshift knife accordingly.
#### Testing

Loaded up game, tested they work right.

#### Additional context

[] swords_and_blades
[]

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
Oh no, I messed up. Meant to change the M-Knife.

**Comment by Night-Pryanik** [comment]
> How'd I revert this extremely dumb mistake?

Just return previous values to spike.

**Comment by Xenomorph-III** [comment]
If there are any problems, please tell me.

**Comment by Hirmuolio** [comment]
You accidentally removed the "[" from the beginning of the json files.

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[
+
```
Comment (AMurkin) [inline data/json/items/melee/swords_and_blades.json]:
```suggestion
[
```

File: data/json/recipes/recipe_others.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
-[
+
```
Comment (AMurkin) [inline data/json/recipes/recipe_others.json]:
```suggestion
[
```

File: data/json/recipes/recipe_others.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+
```
Comment (AMurkin) [inline data/json/recipes/recipe_others.json]:
```suggestion
```

File: data/json/recipes/recipe_others.json [hunk 4]
```diff
@@ -1369,7 +1369,7 @@
     "reversible": true,
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "steel_chunk", 2 ], [ "scrap", 5 ] ] ]
+    "components": [ [ "steel_chunk", 1 ] ]
```
Comment (kevingranade) [inline data/json/recipes/recipe_others.json]:
```suggestion
    "components": [ [ [ "steel_chunk", 1 ] ] ]
```

**Pull request merged**